src/test/java/tempest/ui/pages/HomePageTest.java

```diff
@@ -60,4 +60,22 @@ public void chartViewButton_Sessions() {
 
         assertEquals(PageNames.CHART_VIEW, manager.getCurrentCard());
     }
+
+    @Test
+    public void manageGoalsButtonNoModules(){
```

Comment:
A picky minor change - but for these 2 tests just add an underscore before Modules and NoModules